Panels/SongEditor/TrackName.gd

```diff
@@ -0,0 +1,22 @@
+extends ToolButton
+
+onready var delete_confirm: ConfirmationDialog = $DeleteConfirm
+
+var segments: HBoxContainer
+
+func set_instrument(icon: Texture, instrument_name: String, segments: HBoxContainer) -> void:
+	self.icon = icon
+	self.text = (instrument_name if instrument_name.length() <= 20
+								else instrument_name.substr(0, 17) + "...")
```

Comment:
Ternary If here may be unnecessary, go for a normal if :)

Reply:
Normal if it is.